packages/cli/lib/lib/webpack/render-html-plugin.js

```diff
@@ -109,19 +108,18 @@ module.exports = async function (config) {
 	if (config.prerenderUrls) {
 		if (existsSync(resolve(cwd, config.prerenderUrls))) {
 			try {
-				let result = require(resolve(cwd, config.prerenderUrls));
+				let result = esmImport(resolve(cwd, config.prerenderUrls));
+
 				if (typeof result.default !== 'undefined') {
-					result = result.default();
+					result = result.default;
 				}
 				if (typeof result === 'function') {
-					info(`Fetching URLs from ${config.prerenderUrls}`);
 					result = await result();
-					info(`Fetched URLs from ${config.prerenderUrls}`);
 				}
 				if (typeof result === 'string') {
 					result = JSON.parse(result);
 				}
-				if (result instanceof Array) {
+				if (Array.isArray(result)) {
```

Comment:
Not sure why this stopped working, but probably shouldn't be using `instanceof` anyhow